bitnami/keycloak/values.yaml

```diff
@@ -58,7 +58,7 @@ extraDeploy: []
 image:
```

Comment:
Please remove this change from the PR since it belongs to keycloak

Reply:
Done.Please review it again.Thank you^_^